README.hpcgap.md

```diff
@@ -0,0 +1,14 @@
+# HPC-GAP
+
+The beta release of GAP 4.9 includes experimental code to support multithreaded
```

Comment:
"the beta release"? But the final release also will include it, no?

Also, should we really mention a version here?

So perhaps just say: "GAP includes experimental code..."